cal/cal/models.py

```diff
@@ -28,6 +28,7 @@ class Profile(models.Model):
     main_calendar = models.ForeignKey("GCalendar", null=True)
     authed = models.BooleanField(default=False, help_text="If the user's oauth credentials are currently valid")
     analysis_start = models.DateTimeField(null=True, blank=True, help_text="When the analysis of the user's calendar will start")
+    first_time_authed = models.BooleanField(default=False, help_text="If it's the first time the user's oauth credentials are authenticated")
```

Comment:
Also be wary of asking other people to migrate before your branch has been pushed into master. If you push your migrations (the generated code) their local environments will see that the migration has not been run and attempt to run it, but find that the field `first_time_authed` already exists.

Why I say "pushed into master" is because that's when all the migrations have been assumed to be safe and finalized, so our production should run the migration, as well as the other devs.

Anyone who has run this migration already will likely need to enter their mysql shell and drop the column created by this field. Dw about it though, we can figure it out if it comes up